plutus-benchmark/lists/test/Lookup/Spec.hs

```diff
@@ -1,20 +1,20 @@
 module Lookup.Spec (tests) where
 
 import Test.Tasty
-import Test.Tasty.Extras (TestNested, runTestGroupNestedGhc)
+import Test.Tasty.Extras (TestNested, runTestNested, testNestedGhc)
 
 import PlutusBenchmark.Lists.Lookup.Compiled qualified as Compiled
 
 import PlutusTx.Test qualified as Tx
 
 -- Make a set of golden tests with results stored in a given subdirectory
 -- inside a subdirectory determined by the GHC version.
-testGroupGhcIn :: [FilePath] -> [TestNested] -> TestTree
-testGroupGhcIn dir = runTestGroupNestedGhc (["lists", "test"] ++ dir)
+runTestGhc :: [FilePath] -> [TestNested] -> TestTree
+runTestGhc path = runTestNested (["lists", "test"] ++ path) . pure . testNestedGhc
```

Comment:
I know this is more wordy than it used to be, but here's my reasoning: `TestNested` stuff should have the same ergonomics as `TestTree` and if you want to add an indirection/label in the latter case you use `testGroup name . pure`, so here we're doing the same thing except it's `runTestNested path . pure` (below you'll find examples of `testNested name . pure`). It kinda feels weird to have a special runner when there's "GHC" at the end of the path, given that this is easy to create on the fly without adding more stuff to the API.

This could also be written as 

```haskell
runTestNestedM (["lists", "test"] ++ path) . testNestedGhc
```

but I decided to make it the `pure` version for the reader to learn that they can do it here and similarly with the regular `testGroup`.

Also before the API would nudge you into always having "GHC" at the end of the path, now you're forced to make a choice yourself.